src/ray/core_worker/core_worker.cc

```diff
@@ -426,15 +426,34 @@ CoreWorker::CoreWorker(const CoreWorkerOptions &options, const WorkerID &worker_
                 << rpc_address_.port() << ", worker ID " << worker_context_.GetWorkerID()
                 << ", raylet " << local_raylet_id;
 
+  // Begin to get gcs server address from raylet.
+  gcs_server_address_updater_ = std::unique_ptr<GcsServerAddressUpdater>(
+      new GcsServerAddressUpdater(options_.raylet_ip_address, options_.node_manager_port,
+                                  [this](std::string ip, int port) {
+                                    absl::MutexLock lock(&gcs_server_address_mutex_);
+                                    gcs_server_address_.first = ip;
+                                    gcs_server_address_.second = port;
+                                  }));
```

Comment:
Nit: We're on C++14 now, so we can use the more idiomatic `std::make_unique`:
```suggestion
  gcs_server_address_updater_ = std::make_unique<GcsServerAddressUpdater>(
      options_.raylet_ip_address, options_.node_manager_port,
      [this](std::string ip, int port) {
        absl::MutexLock lock(&gcs_server_address_mutex_);
        gcs_server_address_.first = ip;
        gcs_server_address_.second = port;
      }));
```

Reply:
No problem :)